cmdline/configfile.md

```diff
@@ -59,7 +59,7 @@ it easier to read.
 A command line option that takes an argument must have its argument provided on
 the SAME LINE as the option. 
 
-#### Example 1
+#### Example 1 {#option-with-arguments-on-same-line-example-one}
```

Comment:
Why are these extra text (anhors?) added?

Reply:
There are multiple headings titled 'Example 1' in this file, so the default ids for these headings are duplicates

Having duplicates breaks things like links from the table of contents (and would break links from the index, if any of the index words wanted to link to these sections)

It is likely possible to update the `uni.pl` and `mkindex.pl` scripts to de-dupe these automatically, but in lieu of that we can specify a custom section id in the *.md files directly to de-dupe where needed.

Reply:
I landed #406 which removed all headings called "example", as I believe they were not very well placed there and should be done better.